### PR TITLE
Add suppress annotation to fix libs error in the some build.gradle.kts

### PR DESCRIPTION
**Pull request by SaeedNoshadi89**
This is related to #34.
I used suppress annotation to fix the error red line in the some gradle files `libs` code.

### CODE REVIEW

File: core-database/build.gradle.kts [hunk 1]
```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@kotlin.Suppress("DSL_SCOPE_VIOLATION")
```
Comment (alexvanyo) [inline core-database/build.gradle.kts:16]: Small nit: On my side, it looks like the `kotlin.` is unnecessary, I think just the plain `Suppress` is clear enough!
```suggestion
@Suppress("DSL_SCOPE_VIOLATION")
```

File: core-datastore/build.gradle.kts [hunk 2]
```diff
@@ -19,6 +19,7 @@ import com.google.protobuf.gradle.generateProtoTasks
 import com.google.protobuf.gradle.protobuf
 import com.google.protobuf.gradle.protoc
 
+@kotlin.Suppress("DSL_SCOPE_VIOLATION")
```
Comment (alexvanyo) [inline core-datastore/build.gradle.kts:22]: Could we add a TODO with the upstream issue to track this? I believe this is the correct one: https://youtrack.jetbrains.com/issue/KTIJ-19369

Something like
```suggestion
// TODO: Remove once https://youtrack.jetbrains.com/issue/KTIJ-19369 is fixed
@kotlin.Suppress("DSL_SCOPE_VIOLATION")
```

**Review by alexvanyo — COMMENTED** [review]
Thank you, this looks great! Glad to be able to avoid those annoying squiggly red lines. Just two minor comments from me!

File: core-datastore/build.gradle.kts (hunk 2 above)
Comment (SaeedNoshadi89) [reply core-datastore/build.gradle.kts:22]: It seems great. Thank you

**Review by alexvanyo — APPROVED** [review]
Sorry for the delay, this looks great!

**Pull request merged**